src/app.js

```diff
@@ -23,6 +25,18 @@ app.use(
 app.use(passport.initialize());
 app.use(passport.session());
 app.use('/auth', authRouter);
+const { newsletter, serverStatus } = require('./utils/cron');
+
+const allProjects = async () => {
+  const projects = await Project.find();
+  const titles = [];
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < projects.length; i++) {
+    titles.push(projects[i].title);
+  }
+  newsletter(titles.join());
+};
+allProjects();
```

Comment:
Instead of crowding the app.js file you can implement a function in `cron.js` file and call it here.

Example cron.js
```
function initializeCronTasks() {
  cron.schedule(time , () => {
    _Your code_
  }

  cron.schedule(time , () => {
    _Your code_
  }
}

module.exports = initializeCronTasks
```

Call it in app.js:
```
const initializeCronTasks = require('./utils/cron')

initializeCronTasks()
```